components/atom/loading/src/index.js

```diff
@@ -0,0 +1,11 @@
+import PropTypes from 'prop-types'
+
+const AtomLoading = ({role = 'status', ...props}) => {
+  return <div role="status" {...props} className="sui-AtomLoading" />
```

Comment:
We have defined a default value for `role` prop but we are not using it inside, right?

```suggestion
  return <div role={role} {...props} className="sui-AtomLoading" />
```